### PR TITLE
[ATK] Installing .pc files

### PR DESCRIPTION
**Pull request by faserg1**
Prepare and install atk.pc file.
This pull request is a part of #9564 solution.

### CODE REVIEW

File: ports/atk/portfile.cmake [hunk 1]
```diff
@@ -15,7 +15,11 @@ vcpkg_extract_source_archive_ex(
         fix-linux-config.patch
 )
 
-file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH})
+set(ATK_LIB_SUFFIX 2.0)
```
Comment (JackBoosY) [inline ports/atk/portfile.cmake:18]: Should be `1.0`?

File: ports/atk/portfile.cmake [hunk 2]
```diff
@@ -15,7 +15,11 @@ vcpkg_extract_source_archive_ex(
...
+set(ATK_DLL_SUFFIX 2)
```
Comment (JackBoosY) [inline ports/atk/portfile.cmake:19]: Should be `1`?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/atk/portfile.cmake (hunk 1 above)
Comment (faserg1) [reply ports/atk/portfile.cmake:18]: Should be 2.0
There is new version already, and for windows, as I understood, used different ATK_API_VERSION. I'll test it locally and update soon.

Comment (JackBoosY) [reply ports/atk/portfile.cmake:18]: I noticed you didn't update the version before?

Comment (faserg1) [reply ports/atk/portfile.cmake:18]: What do you mean?

Comment (JackBoosY) [reply ports/atk/portfile.cmake:18]: Oh, never mind, the version in master branch is incorrect.

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Please bump the port version. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).

**Comment by faserg1** [comment]
ATK compiles. Now failed GTK for some reason. Very strange.

**Comment by JackBoosY** [comment]
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
ATK_LIBRARY
    linked by target "gtk-3" in directory D:/buildtrees/gtk/src/gtk+-3-b723a7b846.clean
    linked by target "gdk-3" in directory D:/buildtrees/gtk/src/gtk+-3-b723a7b846.clean
    linked by target "gailutil-3" in directory D:/buildtrees/gtk/src/gtk+-3-b723a7b846.clean

CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_INSTALL_BINDIR
    CMAKE_INSTALL_LIBDIR


CMake Generate step failed.  Build files cannot be regenerated correctly.
```
It seems that because you fixed ATK's pkgconfig, gtk uses the pkgconfig internally to find atk, resulting this error.

**Comment by faserg1** [comment]
@JackBoosY Should I bump port-version for gtk if I update CMakeLists.txt in GTK port folder? There was used old 1.0 version

**Comment by JackBoosY** [comment]
@faserg1 No, since this is an error using atk's pkgconfig, I think you should fix pkgconfig.

**Comment by faserg1** [comment]
@JackBoosY I just do not understand, why in CMakeLists.txt and portfile version is 1.0, if used sources of >2.
Again, I can modify this line:
https://github.com/microsoft/vcpkg/blob/master/ports/gtk/CMakeLists.txt#L9
to ver2
Or return everything back, like there is ATK ver1

**Comment by JackBoosY** [comment]
Seems that never changes since atk was added to vcpkg, see [log](https://github.com/microsoft/vcpkg/commit/41942fa9b75e30a155abf238bad737f953ada610).

**Comment by faserg1** [comment]
@JackBoosY So leave it as is? Ver1, right?

**Comment by JackBoosY** [comment]
@faserg1 I think your changes is right.

**Comment by faserg1** [comment]
@JackBoosY So if update to ver2 I need to patch gtk CMakeFiles.txt
https://github.com/microsoft/vcpkg/blob/c303dd024e2dc03a0665ddffbb12a1077933e649/ports/gtk/CMakeLists.txt#L9
That what I am talking about =)

**Comment by JackBoosY** [comment]
@faserg1 Yep, you're right, I think that's the reason why this regression happened!

**Comment by faserg1** [comment]
@JackBoosY Ah, no, there is API version of atk, it's ver1. So I need just return back. It's so confusing...
If just look at atk source (`atk.pc.in`), there will be ver1.
```Libs: -L${libdir} -latk-@ATK_API_VERSION@
Cflags: -I${includedir}/atk-1.0
```

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**